hardware/neopixel/package.json

```diff
@@ -1,22 +1,25 @@
 {
-    "name"          : "node-red-node-pi-neopixel",
-    "version"       : "0.0.16",
-    "description"   : "A Node-RED node to output to a neopixel (ws2812) string of LEDS from a Raspberry Pi.",
-    "dependencies"  : {
-    },
-    "repository" : {
-        "type":"git",
-        "url":"https://github.com/node-red/node-red-nodes/tree/master/hardware/neopixel"
+    "name": "node-red-node-pi-neopixel",
+    "version": "0.0.17.sw2",
```

Comment:
.sw2 ?

Reply:
I never know what do do re incrementing  someone elses code (as they might have uncommitted changes) but I'll just bump it up to plain 0.0.17

Reply:
to 0.0.17